gen/internal/tests/thrift/set_to_slice.thrift

```diff
@@ -0,0 +1,26 @@
+typedef set<string> (go.type = "slice") StringList
+typedef set<Foo> (go.type = "slice") FooList
+typedef StringList MyStringList
+typedef MyStringList AnotherStringList
+
+typedef set<set<string> (go.type = "slice")> (go.type = "slice") StringListLsit
+
+struct Foo {
+    1: required string stringField
+}
+
+struct Bar {
```

Comment:
This is a fairly thorough example. Thanks!

Would you mind adding these types to the `gen/quick_test.go` test suite? It'll exercise a lot of the standard code paths in these types. That'll help that little coverage drop as well.

Reply:
Done